cypress/integration/page.spec.js

```diff
@@ -1036,7 +1114,7 @@
       page.spyOnCommentsPost();
 
       newCommentForm.nameInput().type('Some name');
-      newCommentForm.bodyArea().type('Some comment body')
+      newCommentForm.bodyArea().type('Some comment body');
       newCommentForm.submitButton().click();
 
       cy.get('@commentsPost').should('not.be.called');
```

Comment:
The method `cy.get('@commentsPost').should('not.be.called')` should be `cy.get('@commentsPost').should('not.have.been.called')` to correctly assert that the spy was not called.